v1.0/docs/admin/cluster-components.md

```diff
@@ -1,5 +1,6 @@
 ---
 layout: docwithnav
+title: Kubernetes Cluster Admin Guide: Cluster Components
```

Comment:
This particular document is in queue for an edit and titles like these will be scrubbed for consistency.